ibis/backends/pandas/execution/util.py

```diff
@@ -107,12 +108,16 @@ def coerce_to_output(
     result_name = getattr(expr, '_name', None)
 
     if isinstance(expr, (ir.DestructColumn, ir.StructColumn)):
-        return ibis.util.coerce_to_dataframe(result, expr.type().names)
+        return coerce_to_dataframe(
+            result, expr.type().names, expr.type().types
+        )
     elif isinstance(expr, (ir.DestructScalar, ir.StructScalar)):
         # Here there are two cases, if this is groupby aggregate,
         # then the result e a Series of tuple/list, or
         # if this is non grouped aggregate, then the result
-        return ibis.util.coerce_to_dataframe(result, expr.type().names)
+        return coerce_to_dataframe(
+            result, expr.type().names, expr.type().types
```

Comment:
just passing `expr.type()`?

Reply:
Yep can do that, I saw that dask defines a similar function so wanted to be consistent with it, but probably doesn't matter too much: https://github.com/ibis-project/ibis/blob/master/ibis/backends/dask/execution/util.py#L153